### PR TITLE
Format updated for download example button

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to update the format of the download example button in all the C2C source docs.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-271

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/okta-source.md [hunk 1]
```diff
@@ -90,13 +90,13 @@ eventTypes | String | No | `null` | Comma separated list of events to collect. R
 
 <CodeBlock language="json">{MyComponentSource}</CodeBlock>
 
-[Download example](/files/c2c/okta/example.json)
+<a href="/files/c2c/okta/example.tf" target="_blank">Download example</a>
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/okta-source.md:93]:
```suggestion
<a href="/files/c2c/okta/example.json" target="_blank">Download example</a>
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/webex-source.md [hunk 2]
```diff
@@ -116,13 +116,13 @@ Sources can be configured using UTF-8 encoded JSON files with the Collector Ma
...
 
-[Download example](/files/c2c/webex/example.json)
+<a href="/files/c2c/webex/example.json" target="_blank">Download example</a>
 
 ### Terraform example
 
 <CodeBlock language="json">{TerraformExample}</CodeBlock>
 
-[Download example](/files/c2c/webex/example.tf)
+<a href="/files/c2c/webex/example.tf)" target="_blank">Download example</a>
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/webex-source.md:125]:
```suggestion
<a href="/files/c2c/webex/example.tf" target="_blank">Download example</a>
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/workday-source.md [hunk 3]
```diff
@@ -106,13 +106,13 @@ Sources can be configured using UTF-8 encoded JSON files with the Collector Ma
 
 <CodeBlock language="json">{MyComponentSource}</CodeBlock>
 
-[Download example](/files/c2c/workday/example.json)
+<a href="/files/c2c/workday/example.json)" target="_blank">Download example</a>
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/workday-source.md:109]:
```suggestion
<a href="/files/c2c/workday/example.json" target="_blank">Download example</a>
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/workday-source.md [hunk 4]
```diff
@@ -106,13 +106,13 @@ Sources can be configured using UTF-8 encoded JSON files with the Collector Ma
...
 
 ### Terraform example
 
 <CodeBlock language="json">{TerraformExample}</CodeBlock>
 
-[Download example](/files/c2c/workday/example.tf)
+<a href="/files/c2c/workday/example.tf)" target="_blank">Download example</a>
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/workday-source.md:115]:
```suggestion
<a href="/files/c2c/workday/example.tf" target="_blank">Download example</a>
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/zero-networks-segment-source.md [hunk 5]
```diff
@@ -88,13 +88,13 @@ Sources can be configured using UTF-8 encoded JSON files with the Collector Ma
...
 
-[Download example](/files/c2c/zero-networks-segment/example.json)
+<a href="/files/c2c/zero-networks-segment/example.json)" target="_blank">Download example</a>
 
 ### Terraform example
 
 <CodeBlock language="json">{TerraformExample}</CodeBlock>
 
-[Download example](/files/c2c/zero-networks-segment/example.tf)
+<a href="/files/c2c/zero-networks-segment/example.tf)" target="_blank">Download example</a>
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/zero-networks-segment-source.md:97]:
```suggestion
<a href="/files/c2c/zero-networks-segment/example.tf" target="_blank">Download example</a>
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
It looks great, @JV0812! I found a few things you need to fix. I also found and updated some other download links (see commit "[Fix a few more instances](https://github.com/SumoLogic/sumologic-documentation/pull/4199/commits/7450511f0fa4166245753849ac5be2671416e97a)"). 

I don't think you need code anymore at the top of the articles to import the files. It's not causing any problems, but you might want to remove it. For example:
>import MyComponentSource from '!!raw-loader!/files/c2c/airtable/example.json';
>import TerraformExample from '!!raw-loader!/files/c2c/airtable/example.tf';

**Comment by JV0812** [comment]
@jpipkin1 I thought these will be required to the <codeblocks> to display the json or .tf example. Let me cross check it and if not required i will remove this too.

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**